lib/redcord/relation.rb

```diff
@@ -67,24 +66,93 @@ def count
     redis.find_by_attr_count(model.model_key, query_conditions)
   end
 
-  sig { returns(T::Array[T.untyped]) }
-  def to_a
-    execute_query
-  end
+  delegate(
```

Comment:
Just wondering how you came up with this extensive list?

Reply:
I went through https://ruby-doc.org/core-2.7.0/Array.html#method-i-7C and inspected existing methods on an array (with active support / array loaded). I selected most related methods to delegate.